__tests__/api.test.js

```diff
@@ -3,6 +3,7 @@ const seed = require("../db/seeds/seed");
 const testData = require("../db/data/test-data");
 const request = require("supertest");
 const app = require("../app.js");
+const users = require("../db/data/test-data/users");
```

Comment:
Why are you requiring the raw users data in here?

Reply:
VS Code auto imported for some reason, I have removed it now